packages/cli/README.md

```diff
@@ -1 +1,246 @@
-# cli
+# Bucket CLI
+
+Command-line interface for interacting with Bucket services. The CLI allows you to manage apps, features, authentication, and generate TypeScript types for your Bucket features. With this tool, you can streamline your feature flagging workflow directly from your terminal.
+
+## Installation
+
+You can install the Bucket CLI either locally to your project or globally on your system.
+
+### Local Installation (Recommended)
+
+Local installation is recommended for project-specific usage and ensures consistent CLI versions across your team.
+
+```bash
+# npm
+npm install --save-dev @bucketco/cli
+
+# yarn
+yarn add --dev @bucketco/cli
+
+# pnpm
+pnpm add --save-dev @bucketco/cli
+```
+
+When installed locally, use the CLI with npx:
+
+```bash
+# npm
+npx bucket <command>
+```
+
+or as a `package.json` script
+
+```json
+{
+  "scripts": {
+    "bucket": "bucket"
```

Comment:
this seems awkward... If you're using `yarn` at least you can just do `yarn bucket` without this, but it does look like `npm` doesn't give you that. Hmmm

Reply:
Removed this and wrote the alternatives.